src/src/main.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class main {
+    public static void main(String[] args) {
+       Scanner scanner = new Scanner(System.in);
+       boolean hasQ=scanner.hasNext("q");
+       int[] numbers= new int[0];
+       int min=100;
```

Comment:
das sind keine guten intialwerte. Was wenn dein array nur zahlen über 100 hat? dann kommt 100 als minimum raus obwohls nicht in der liste war. 

das selbe wenn das array nur negative zahlen hat. dann kommt 0 als maximum obwohls nicht stimmt.